src/content/index.mdx

```diff
@@ -1,55 +1,73 @@
-The algorithms are built on top of recipes that encode configurations and hyperparamers, enabling easy integration to common frameworks with only a few lines of code.
+Simply put, deep learning deployments no longer need to choose between the performance of GPUs and simplicty of software!
 
-- [SparseZoo](/products/sparsezoo) stores dense and presparsified models/recipes ready for deployment, sparsification, and fine-tuning onto your data.
-SparseZoo stubs enable you to reference any model on the SparseZoo in a convenient and identifiable way.
-They are found throughout the documentation and the model pages on the [SparseZoo website](https://sparsezoo.neuralmagic.com).
+## Neural Magic Platform
 
-## Starting Points
+The Neural Magic Platform enables two major workflows.
 
-<LinkCards>
-  <LinkCard href="/get-started/try-a-model" heading="Get Started">
-    A step-by-step introduction into basic Deep Sparse Platform features.
-  </LinkCard>
+### 1. Optimize a Model for Inference
 
-  <LinkCard href="/use-cases/natural-language-processing/question-answering" heading="Use Cases">
-    A non-exhaustive list of scenarios the Deep Sparse Platform can help with.
-  </LinkCard>
+**SparseML** and **SparseZoo** work together to optimize models for inference with
+techniques like pruning and quantization (which we call "sparsity").
 
-  <LinkCard href="/user-guide/sparsification" heading="User Guide">
-    Study the detailed inner workings of the Deep Sparse Platform in its user guide.
-  </LinkCard>
+- [SparseML](/products/sparseml) is an open-source library that extends PyTorch and TensorFlow to simplify the process of 
+  applying sparsity algorithms. Via simple CLI scipts or 5 lines of code, users can sparsify any model from scratch 
+  or sparse transfer learn from pre-sparsified versions of foundation models like ResNet, YOLOv5, or BERT.
 
-  <LinkCard href="/products/deepsparse" heading="Products">
-    See all of the Deep Sparse Platform's products and available commands and APIs.
-  </LinkCard>
-</LinkCards>
+- [SparseZoo](/products/sparsezoo) is an open-source repository of pre-sparsified models
+  (for example, sparse ResNet-50 has 95% of weights set to 0 while maintaining 99% of the baseline accuracy). SparseZoo is integrated with 
+  SparseML, making it trival for users to fine-tune from sparse model (which we call "Sparse Transfer Learning") onto their data.
 
+### 2. Deploy a Model on CPUs
+
+**DeepSparse** runs inference-optimized sparse models with GPU-class performance on CPUs.
+
+- [DeepSparse](/products/deepsparse) is a deep learning deployment SDK, which includes an inference runtime wrapped with APIs.
+  With inference-optimized sparse models, the runtime offers GPU-class performance on commodity hardware. 
+  The APIs enable users to integrate a model into an application and to monitor the performance of models running in production.
+
+## Docs Content
+
+The documentation is organized into five sections:
```

Comment:
```suggestion
The documentation is organized into several sections:
```

Reply:
When possible, it's best not to be specific because the section number can change and then this has to be updated. Generic will cover all these cases.